docs/general/testing/macos.md

```diff
@@ -0,0 +1,71 @@
+---
+uid: testing-macos
+title: macOS
+description: Test unstable builds on macOS.
+sidebar_position: 2
+---
+
+# Test unstable builds on macOS
+
+Testing unstable builds requires you to be confident with the command line, or at least not afraid of it.
+
+## Get Unstable Server
+
+Unlike the release version, the unstable builds for macOS will not come with a packaged DMG or application. Instead, you have to run them from the command line.
+
+The latest unstable builds for packaged jellyfin can be obtained here:
+
+- [Intel Macs](https://repo.jellyfin.org/?path=/server/macos/latest-unstable/amd64)
+- [Apple Silicon Macs](https://repo.jellyfin.org/?path=/server/macos/latest-unstable/arm64)
+
+There is no difference in content between the `tar.gz` and the `tar.xz` archives; they are only compressed in different formats. Either one will work.
+
+## Get Jellyfin FFmpeg
+
+The unstable version doesn't come with FFmpeg by default, so you need to obtain FFmpeg manually.
+
+Jellyfin's custom FFmpeg is available [here](https://repo.jellyfin.org/?path=/ffmpeg/macos).
+
+Select the version corresponding to the unstable version of the server.
+
+Intel Macs should download the `AMD64` binaries.
+
+Apple Silicon Macs should download the `ARM64` binaries.
```

Comment:
I see the difference between AMD64 and ARM64 mentioned twice. Maybe it would be cleaner to mention it once on the top?

Reply:
This is actually the first time it appears as human-readable form. The former one is in a link and the user will just see "Intel" and "Apple Silicon"

Reply:
What I meant was put the Apple Silicon -> ARM64, Intel -> AMD64 on top and make both the upper level links